src/twister2/report/test_results_plugin.py

```diff
@@ -206,21 +208,23 @@ def _generate_report(self, session: pytest.Session) -> dict:
             subtests_fail_count += sum(1 for st in result.subtests if st['status'] == Status.FAILED)
             subtests_skip_count += sum(1 for st in result.subtests if st['status'] == Status.SKIPPED)
 
-            test = dict(
-                suite_name=get_suite_name(item),
+            testsuites = dict(
```

Comment:
it would be easier to compare T1 with T2, when we keep the order of elements: If you run TwisterV1, then you receive:
name. arch, platform, run_id, runnable, retries, status, reason, execution_time